### PR TITLE
Add class description in v1alpha1 catalog data schema

### PR DESCRIPTION
**Pull request by shubham251972**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: schemas/constructs/v1alpha1/catalog_data.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
         "official",
         "verified",
         "community"
+      ],
+      "enumDescriptions": [
+        "Content produced and fully supported by Meshery maintainers. This represents the highest level of support and is considered the most reliable.",
+        "Content produced by partners and verified by Meshery maintainers. While not directly maintained by Meshery, it has undergone a verification process to ensure quality and compatibility.",
```
Comment (leecalcote) [inline schemas/constructs/v1alpha1/catalog_data.json:18]:
```suggestion
        "Content produced by partners and verified by Meshery maintainers. While not directly maintained by Meshery, content has undergone a verification process to ensure quality and compatibility.",
```

File: schema.go [hunk 2]
```diff
@@ -1,7 +1,28 @@
 package schemas
 
-import "embed"
-
-//go:embed schemas/openapi.yml
+import (
```
Comment (leecalcote) [inline schema.go:3]: Is this appropriate for the schemas repo or more so for meshkit repo?

Other golang components that use these schemas will import the schemas as packages and have full access to the content of each schema. Is that right?

I'm trying to understand the need for this utility function.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: schema.go (hunk 2 above)
Comment (shubham251972) [reply schema.go:3]: Yes, this will allow other components to have full access to the content of each schema

**Comment by leecalcote** [comment]
@shubham251972 will you be taking care of this? https://github.com/meshery/schemas/issues/109

**Comment by leecalcote** [comment]
Please see the `capabilities` schema for an example of how to capture enums and their descriptions.

**Pull request closed without merging**